docs/source/graphiql.md

```diff
@@ -60,17 +60,82 @@ app.use(
 If you are using Hapi, GraphiQL can be configured as follows:
 
 ```js
-import { graphiqlHapi } from 'apollo-server-hapi';
-
-server.register({
-  register: graphiqlHapi,
-  options: {
-    path: '/graphiql',
-    graphiqlOptions: {
-      endpointURL: '/graphql',
+const Hapi = require('hapi');
+const { graphqlHapi, graphiqlHapi } = require('apollo-server-hapi');
+const { makeExecutableSchema } = require('graphql-tools');
+
+const HOST = 'localhost';
+const PORT = 3000;
+
+const StartServer = async () => {
+  const server = new Hapi.server({
+    host: HOST,
+    port: PORT,
+  });
+
+  // Some fake data
+  const books = [
+    {
+      title: "Harry Potter and the Sorcerer's stone",
+      author: 'J.K. Rowling',
     },
-  },
-});
+    {
+      title: 'Jurassic Park',
+      author: 'Michael Crichton',
+    },
+  ];
+
+  // The GraphQL schema in string form
+  const typeDefs = `
+    type Query { books: [Book] }
+    type Book { title: String, author: String }
+  `;
+
+  // The resolvers
+  const resolvers = {
+    Query: { books: () => books },
+  };
+
+  // Put together a schema
+  const schema = makeExecutableSchema({
+    typeDefs,
+    resolvers,
+  });
+
+  await server.register({
```

Comment:
Since GraphiQL is an additional endpoint which can be enabled (if desired), it'd be better if this code block didn't repeat the Hapi-setup code (for the non-GraphiQL aspect) which is found elsewhere in the documentation.

Instead, it would be better to just give a brief suggestion to add the `await server.register({...})` code found below on lines 118-126, to the existing code which was demonstrated in the "setup" step for the GraphQL server itself.